### PR TITLE
feat: support json request for ABAC

### PR DESCRIPTION
**Pull request by PokIsemaine**
fix: https://github.com/casbin/casbin/issues/1272

### CODE REVIEW

**Comment by hsluoyz** [comment]
@nengc plz review

File: enforcer.go [hunk 1]
```diff
@@ -17,11 +17,14 @@ package casbin
 import (
 	"errors"
 	"fmt"
+	"regexp"
 	"runtime/debug"
 	"strings"
 	"sync"
 
 	"github.com/Knetic/govaluate"
+	"github.com/tidwall/gjson"
```
Comment (hsluoyz) [inline enforcer.go:26]: sort imports

File: model_test.go [hunk 2]
```diff
@@ -412,6 +412,20 @@ func TestABACModel(t *testing.T) {
 	testEnforce(t, e, "bob", data1, "write", false)
 	testEnforce(t, e, "bob", data2, "read", true)
 	testEnforce(t, e, "bob", data2, "write", true)
+
+	// json request
+	e.EnableAcceptJsonRequest(true)
```
Comment (hsluoyz) [inline model_test.go:417]: Separate test case

**Review by hsluoyz — CHANGES_REQUESTED** [review]

**Review by hsluoyz — APPROVED** [review]

**Comment by nengc** [comment]
@hsluoyz @PokIsemaine Tks guys, It works.

**Pull request merged**